02_Getting_started.jl

```diff
@@ -174,7 +188,46 @@ distance_x_2 = 2 * distance
 # confuse it with an actual shell: special shell syntax like piping won't work.
 #md # Like before, hit <kbd>backspace</kbd> to get back to the Julia prompt.
 
-# !!! challenge "Hello, `shell>` !"
+# !!! challenge "Hello, `shell>` (`pwd` and `cd`) !"
+#md #      Two commonly used shell commands are `pwd` (**p**rint **w**orking **d**irectory) and `cd` (**c**hange **d**irectory).
+#md #      1. Use `pwd` to find out what is your current working directory.
+#md #      2. Type the command `cd` in shell mode, which by default will bring you to your "home directory".
+#md #      3. Use `pwd` again. Did you get a different result from before? Why or why not?
+#     !!! solution
+#md #         ```julia-repl
+#md #         shell> pwd
+#md #         ```
+#md #
+#md #         ```julia-repl
+#md #         shell> cd
+#md #         ```
+#md #
+#md #         ```julia-repl
+#md #         shell> pwd
```

Comment:
I wonder if it would be worth comparing / contrasting with `cd()` and `readdir()` in julia - using the shell is great for quick things, but you can't then assign things to variables or run julia functions on the outputs.

Or maybe that's an added complication that's not worth it at this stage of the lesson

Reply:
We could mention it in an info box or similar

Reply:
but can also be in a different PR